runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -529,6 +532,34 @@ public void focusLost(FocusEvent e)
 				item.add(button, BorderLayout.EAST);
 			}
 
+			if (cid.getType() == File.class)
+			{
+				JFileChooser fileChooser = new JFileChooser();
+				fileChooser.setDialogTitle("Select a custom cursor image");
+				fileChooser.setAcceptAllFileFilterUsed(false);
+				FileNameExtensionFilter filter = new FileNameExtensionFilter("image files", "png", "gif", "jpeg", "jpg", "bmp", "ico", "cur");
```

Comment:
is this all possible cursor files you think?

Reply:
This should not be specific to cursor plugin